Engine.py

```diff
@@ -277,53 +288,63 @@ def query_player_card(self, player_id : int, ignore_if_dead : bool = False) -> i
 
     def translate_action_choice(self, response : str) -> Action:
         """Given an action response, translate it appropriately"""
-        response = response.strip()
+        response = response.strip().lower()
         args = response.split(" ")
         action_name = args[0]
         target = None if len(args) == 1 else int(args[1])
      
-        if action_name.lower() in ["i", "income"]:
+        if action_name in ["i", "income"]:

```

Comment:
would be nice to be able to put all these in to the the Action class itself. 

so it can be like action_name in Income.Commands()

as usual nbd

Reply:
I have an idea, will implement and commit it in a sec to address this